sqlalchemy_ibmi/base.py

```diff
@@ -775,13 +775,18 @@ def dbapi(cls):
     def create_connect_args(self, url):
         opts = url.translate_connect_args(username='user', host='system')
         opts.update(url.query)
-        allowed_opts = {'system', 'user', 'password',
-                        'autocommit', 'readonly', 'timeout',
-                        'database', 'library_list', 'current_schema'}
+        allowed_opts = {'system', 'user', 'password', 'autocommit', 'readonly',
+                        'timeout', 'database', 'use_system_naming',
+                        'library_list', 'current_schema'
+                        }
+
         if allowed_opts < opts.keys():
             raise ValueError("Option entered not valid for "
                              "IBM i Access ODBC Driver")
 
+        name_setting = '1' if opts.get('use_system_naming', 'True') == 'True' else '0'
```

Comment:
Since this is a string, we should probably handle upper/lower case. I think we can make use of [this API](https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool) to do that for us.

It raises a ValueError if it's not one of the allowed values so we'll have to wrap the whole thing in a try/except block. Because of needing the try/except block, might as well ditch opts.get and refer to the attribute directly and instead handle the AttributeError as a second case.